docs/loggregator-design.md

```diff
@@ -60,7 +62,7 @@ provides the interface for container metrics and recent logs calls. Rather
 than using a diode it does this by notifying the system of slow consumers.
 
 ## Syslog Drain Binders
```

Comment:
I think this entire section should just be deleted. The syslog drain binders were removed several versions ago.